File: serialize/java/src/org/Book.java
package org;

import java.io.Serializable;
import java.util.ArrayList;

public class Book implements Serializable {
    public String name;
    ArrayList<Page> p;
    public Book() {
        this.name = "I am just a little book";
        p = new ArrayList<>();
    }
    public Book(String name) {
        this.name = name;
        p = new ArrayList<>();
    }
    public void add(Page p) {
        this.p.add(p);
    }
    public void show() {
        System.out.println(name);
        for (Page p: this.p) {
            System.out.println(p.read());
        }
    }
}

File: serialize/py/Book.py
class Book(object):
    def __init__(self, n = "I am a little book"):
        self.n = n
        self.p = []

    def add(self, p):
        self.p += [p]

    def show(self):
        print(self.n)
        for p in self.p:
            print(p.read())
